prover/circuits/execution/pi_wizard_extraction.go

```diff
@@ -132,6 +132,16 @@ func checkPublicInputs(
 
 	api.AssertIsEqual(bridgeAddress, gnarkFuncInp.L2MessageServiceAddr)
 
+	api.AssertIsEqual(
+		wvc.GetPublicInput(api, publicInput.BaseFee),
+		gnarkFuncInp.BaseFee,
```

Comment:
This will need an update (as for the whole file as the inputs are broken down in limbs now)